### PR TITLE
ACS-2199 fix list parameters

### PR DESCRIPTION
**Pull request by jesty**
In this PR we add CollectionFormat annotation in order to pass list parameters as CSV, not in the multiple format.
Reference: https://github.com/spring-cloud/spring-cloud-netflix/issues/1634

In previous versions, as workaround users can define a RequestInterceptor in order to fix the request:
```
@Component
public class ListToCSVQueryStringRequestInterceptor implements RequestInterceptor {
    @Override
    public void apply(RequestTemplate template) {
        Map<String, Collection<String>> queries = template
            .queries()
            .entrySet()
            .stream()
            .collect(Collectors.toMap(Entry::getKey, this::getValue));
        template.queries(null);
        template.queries(queries);
    }

    private Collection<String> getValue(Entry<String, Collection<String>> entry) {
        Collection<String> value = entry.getValue();
        if (value.size() > 1) {
            return List.of(String.join(",", value));
        }
        return value;
    }
}
```

### CODE REVIEW

File: alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-auth-rest-api/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Add this dependency to your project's POM:
 Add this dependency to your project's build file:
 
 ```groovy
-compile "org.alfresco:alfresco-auth-rest-api:5.1.1"
+compile "org.alfresco:alfresco-auth-rest-api:5.1.2-SNAPSHOT"
```
Comment (jesty) [inline alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-auth-rest-api/README.md:40]: Is the version update exptected?

File: alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-auth-rest-api/README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ mvn package
 
 Then manually install the following JARs:
 
-* target/alfresco-auth-rest-api-5.1.1.jar
+* target/alfresco-auth-rest-api-5.1.2-SNAPSHOT.jar
```
Comment (jesty) [inline alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-auth-rest-api/README.md:51]: as above, Is the version update exptected?

File: alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-core-rest-api/README.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Add this dependency to your project's POM:
 Add this dependency to your project's build file:
 
 ```groovy
-compile "org.alfresco:alfresco-core-rest-api:5.1.1"
+compile "org.alfresco:alfresco-core-rest-api:5.1.2-SNAPSHOT"
```
Comment (jesty) [inline alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-core-rest-api/README.md:40]: as above, is the version update exptected?

File: alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-core-rest-api/README.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ mvn package
 
 Then manually install the following JARs:
 
-* target/alfresco-core-rest-api-5.1.1.jar
+* target/alfresco-core-rest-api-5.1.2-SNAPSHOT.jar
```
Comment (jesty) [inline alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-core-rest-api/README.md:51]: as above, is the version update exptected?

File: alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/src/main/resources/templates/api.mustache [hunk 5]
```diff
@@ -20,6 +20,7 @@ import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RequestPart;
 import org.springframework.web.multipart.MultipartFile;
+import org.springframework.cloud.openfeign.CollectionFormat;
```
Comment (jesty) [inline alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/src/main/resources/templates/api.mustache:23]: Most important change in this PR

File: alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/src/main/resources/templates/api.mustache [hunk 6]
```diff
@@ -35,6 +35,7 @@ import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RequestPart;
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.bind.annotation.CookieValue;
+import org.springframework.cloud.openfeign.CollectionFormat;
```
Comment (jesty) [inline alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/src/main/resources/templates/api.mustache:38]: Most important change in this PR

File: alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-auth-rest-api/README.md (hunk 1 above)
Comment (miguelruizdev) [reply alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/generated/alfresco-auth-rest-api/README.md:40]: It is probably the first time generated with this version, so yeah, it makes sense for it to have the current version, which is `5.1.2-SNAPSHOT`
Same for the other comments.

**Review by miguelruizdev — APPROVED** [review]

**Pull request merged**